c2corg_ui/static/js/feed.js

```diff
@@ -67,9 +68,9 @@ app.FeedController = function(appAuthentication, appApi, appLang, appUrl,  image
 
   /**
    * @type {number}
-   * @export
+   * @private
    */
-  this.nbcolumn = 0;
+  this.nbCols = 0;
```

Comment:
Private properties must have a trailing ``_`` (so that it is obvious they are private). Eg. ``this.nbCols_``